python/ray/tests/test_advanced_3.py

```diff
@@ -89,6 +89,15 @@ def f(x):
     attempt_to_load_balance(f, [x], 100, num_nodes, 25)
 
 
+def wait_for_num_actors(num_actors, timeout=10):
```

Comment:
Could define this inline since it's only used once.

Reply:
Given that the others are also outside, it is probably better to keep it consistent :)